themes/cold-tavern/main.css

```diff
@@ -42,3 +42,29 @@ body {
 .content strong {
   color: white;
 }
+
+@media screen and (max-width: 850px) {
+  .content {
+    width: 90%;
+    margin-left: 30px;
+  }
+}
+@media screen and (max-width: 768px) {
+  .content {
+    width: 100%;
+    margin-left: 0px;
+  }
+  .lang-sh {
+    width: 62.5%;
+
+    font-size: 0.6em;
+  }
```

Comment:
Did you have problems for Serbo-Croatian?

Reply:
I don't understand ur question?

Reply:
Sorry, was just wondering what the `.lang-sh` selector is for?